index.html

```diff
@@ -28,7 +28,136 @@ <h1>I’m Mariane Glad to see you!</h1>
+           </ul>
+           
+           <button class="card_see" type="button"> See Project</button>
+        </div>
+        
+        <div class="card"><img src="./images/SnapshootPortfolio(2).png" alt="navigation icon" class="snapshoot">
+            <h1>Tonic</h1>
+            
+            <div class="sec">
+               <p>CANOPY</p>
+               <span class="dot"></span>
+               <p class="sec_color1">Back End Dev</p>
+               <span class="dot"></span>
+               <p class="sec_color2">2015</p>
+           </div>
+           
+           <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           <ul class="list">
+               <li class="box"><button type="button">html</button></li>
+               <li class="box"><button type="button">css</button></li>
+               <li class="box"><button type="button">javaScript</button></li>
+           </ul>
+           
+           <button class="card_see" type="button">See Project</button>
+        </div>
+
+        <div class="card">
+            <img src="./images/SnapshootPortfolio(3).png" alt="navigation icon" class="snapshoot">
+             <h1>Multi-Post Stories</h1>
+             
+             <div class="sec">
+                <p>CANOPY</p>
+                <span class="dot"></span>
+                <p class="sec_color1">Back End Dev</p>
+                <span class="dot"></span>
+                <p class="sec_color2">2015</p>
+            </div>
+            
+            <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list">
+                <li class="box"><button type="button">html</button></li>
+                <li class="box"><button type="button">css</button></li>
+                <li class="box"><button type="button">javaScript</button></li>
+            </ul>
+            
+            <button class="card_see" type="button">See Project</button>
+        </div>
+    </section>
+
+    <section class="card aboutme">
+        <h1>About Myself</h1>
+        <p class="para">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="">LET'S CONNECT</a>
+        <ul class="icons">
+            <li class="icons_list"><img src="./images/Vector-4.png" alt="navigation icon" ></li>
+            <li class="icons_list"><img src="./images/Vector-2.png" alt="navigation icon"></li>
+            <li class="icons_list"><img src="./images/Vector-1.png" alt="navigation icon"></li>
+            <li class="icons_list"><img src="./images/Vector-3.png" alt="navigation icon"></li>
+            <li class="icons_list"><img src="./images/Vector.png" alt="navigation icon"></li>
+        </ul>
+       <button class="btn" type="button">Get my resume</button>
```

Comment:
- [ ] Please kindly make sure the same font used in the original design is used here also
- [ ] Please ensure that there is enough padded space at the top, just before the `About Myself` heading
- [ ] Please ensure that the button style reflects the different states(hover, pressed and inactive)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164948574-44976135-ac89-4570-89a8-095d2697f673.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164948586-9f1f4a1f-b9f4-4caf-a1ee-b2ef7b849315.png)